packages/react-docs/pages/components/tooltip.mdx

```diff
@@ -12,86 +12,219 @@ import { Tooltip } from '@tonic-ui/react';
 
 ## Usage
 
-### Controlled and uncontrolled tooltips
+By default, you have to pass a single React element child to the `Tooltip` component.
+
+```jsx disabled
+<Tooltip label="This is a tooltip">
+  <Text display="inline-block">This is the text content</Text>
+</Tooltip>
+```
+
+If you need to pass more than one child element or non-element children, wrap them in an element or pass the `shouldWrapChildren` prop.
+
+```jsx disabled
+<Tooltip label="This is a tooltip">
+  <>This is the text content</>
+</Tooltip>
+
+<Tooltip label="This is a tooltip" shouldWrapChildren>
+  This is the text content
+</Tooltip>
+
+<Tooltip label="This is a tooltip" shouldWrapChildren>
+  <Icon icon="menu" height="5x" mr="2x" />
+  <Text>This is the text content</Text>
+</Tooltip>
+```
+
+### Controlled tooltip
 
 Pass `isOpen` to the `Tooltip` component to control the state of the tooltip.
 
 ```jsx
-<Tooltip label="This is a controlled tooltip" isOpen>
-  <Text>Tooltip anchor</Text>
+function Example() {
+  const [on, toggle] = useToggle(false);
+
+  return (
+    <>
+      <Box mb="4x">
+        <Switch checked={on} onChange={toggle} />
+      </Box>
+      <Tooltip
+        label="This is a controlled tooltip"
+        isOpen={on}
+      >
+        <Text display="inline-block">Tooltip anchor</Text>
+      </Tooltip>
+    </>
+  );
+}
+```
+
+### Uncontrolled tooltip
+
+Tooltip is uncontrolled by default. You can set `defaultIsOpen` to `true` to have the tooltip open for the first render.
+
+```jsx
+<Tooltip
+  label="This is an uncontrolled tooltip"
+  defaultIsOpen={true}
+>
+  <Text display="inline-block">Tooltip anchor</Text>
 </Tooltip>
 ```
 
-Set `defaultIsOpen` to `true` to have the tooltip open by default.
+### Disabled tooltip
+
+Pass `disabled` to the `Tooltip` component to disable the tooltip.
 
 ```jsx
-<Tooltip label="This is an uncontrolled tooltip" defaultIsOpen>
-  <Text>Tooltip anchor</Text>
+<Tooltip label="Search" disabled>
+  <Icon icon="search-o" />
+</Tooltip>
```

Comment:
Is these a solid code for reference?
Coz I could not get the 'disabled' prop work in the corresponding console.
And it seems conflict with 'isOpen'.

Reply:
@light049 There was a build error in https://github.com/trendmicro-frontend/tonic-ui/pull/555/commits/edaf4e375bbfef4d2c713262c408c2a8ef053897 that caused the demo site not updated. It will be fixed in https://github.com/trendmicro-frontend/tonic-ui/pull/555/commits/4b54ac3cfde1cc4e52d63c2439a468736b6a1722

Reply:
Indeed there existed a conflict between `disabled` and `isOpen` props. Let me check and get back to you for a solution.

Reply:
Chakra's Tooltip component also has the same issue https://chakra-ui.com/docs/components/overlay/tooltip

![image](https://user-images.githubusercontent.com/447801/162550853-971dc8b1-bddd-418e-8250-d3e799b9da98.png)

Reply:
> Indeed there existed a conflict between `disabled` and `isOpen` props. Let me check and get back to you for a solution.

The tonic-ui docs for Tooltip doesn't indicate prop 'disabled', and it seems not work neither, so it might be fine just removing corresponding descriptions?

Reply:
I made new changes in https://github.com/trendmicro-frontend/tonic-ui/pull/555/commits/ebb6e775c00c6552a46ca7e6f86a00f9d1574bcc